scripts/build-single-packages.js

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable no-console */
+const fse = require('fs-extra');
+const glob = require('glob');
+
+const root = process.cwd();
+const packageJson = require(`${root}/package.json`);
+const configJson = require(`${root}/${process.argv[process.argv.indexOf('--config') + 1]}`);
```

Comment:
This config file system is interesting, but one drawback IMO is that if you try running the script without a config argument, or the config file isn't where the script expects, it fails with a somewhat confusing node internal error.

What would you think about wrapping this in a try/catch or something that gives a more helpful error message if no config is found?

Reply:
Added a config argument check and error to report if it's not included.